listenbrainz/webserver/templates/user/profile.html

```diff
@@ -12,4 +12,10 @@
 {% block scripts %}
   {{ super() }}
   <script src="{{ get_static_path('recentListens.js') }}" type="text/javascript"></script>
+  <script>
+    // Enable popovers after page load
+    window.addEventListener("load", function(){
```

Comment:
Now that you've replace the popover functionality with a library, this should be removed